### DISCUSSION TITLE
Missing title for some links

### DISCUSSION THREAD

**Discussion by Lost-n-thought** [WebCrateApp/webcrate#57] [Q&A] [answered]
Eg - https://www.udemy.com/course/complete-python-bootcamp/   in webcrate it only shows www.udemy.com whereas in normal Firefox bookmarks it shows - "Python Bootcamps: Learn Python Programming and Code Training | Udemy"

I think it would be more meaningful to have "Python Bootcamps: Learn Python Programming and Code Training | Udemy" as name

     
but 
https://www.amazon.com/All-new-Kindle-Oasis-now-with-adjustable-warm-light/dp/B07F7TLZF4/
is same in both webcrate and firefox

**Comment by BetaHuhn** [comment, marked as answer]
WebCrate tries to parse the website's title and description as soon as you add it, but this can sometimes fail for different reasons. Most of the time, WebCrate's server can't get access to that link or the website is simply not returning a title.

If this does happen, WebCrate will use the website's domain as a fallback title, which is what you see with your Udemy link. You can at any point manually change the title of a link though, just click on it and start typing.

I am planning on adding a feature to the [browser-extension](https://github.com/WebCrateApp/browser-extension) which grabs the title inside the browser directly, which should work far more often than letting the server do it. I can't say when this will be implemented though, I'm hoping I can work on it soon.
